templates/cf-monitord.service.mustache

```diff
@@ -2,11 +2,12 @@
 Description=CFEngine Monitor Daemon
 After=syslog.target
 ConditionPathExists={{{vars.sys.bindir}}}/cf-monitord
+ConditionPathExists={{{vars.sys.inputdir}}}/promises.cf
 PartOf=cfengine3.service
 
 [Service]
 Type=simple
-ExecStart={{{ vars.sys.bindir}}}/cf-monitord --no-fork --verbose
+ExecStart={{{ vars.sys.bindir}}}/cf-monitord --no-fork
```

Comment:
I think this space should be redacted. Instead of `{{{ vars.sys.bindir}}}` `{{{vars.sys.bindir}}}`